FitNesseRoot/FitNesse/UserGuide/TwoMinuteExample/content.txt

```diff
@@ -2,31 +2,30 @@
 
 !define TEST_SYSTEM {slim}
 *!
-[[A One-Minute Description][OneMinuteDescription]]
-
 !1 An Example !-FitNesse-! Test
 If you were testing the division function of a calculator application, you might like to see some examples working. You might want to see what you get back if you ask it to divide 10 by 2. (You might be hoping for a 5!)
 
-In !-FitNesse-!, tests are expressed as tables of '''input''' data and '''expected output''' data. Here is one way to specify a few division tests in !-FitNesse-!:
+In !-FitNesse-!, tests are expressed as tables of '''input''' data and '''expected output''' data. Here is one way to specify a few division tests in !-FitNesse-!:
 
-| eg.Division                         |
+| eg.Division |
 | numerator | denominator | quotient? |
-| 10        | 2           | 5.0       |
-| 12.6      | 3           | 4.2       |
-| 22        | 7           | ~=3.14    |
-| 9         | 3           | <5        |
-| 11        | 2           | 4<_<6     |
-| 100       | 4           | 33        |
+| 10 | 2 | 5.0 |
```

Comment:
This is of course nitpick, but could you realign the table? I find it easier to read the content when the columns are aligned, and possibly lots of people will attempt to edit this page to experiment when starting out.